frontend/src/static/js/main.js

```diff
@@ -260,21 +280,13 @@ window.app = new window.Vue({
     CircleSpinner: window.VueLoadingSpinner.Circle,
   },
   created() {
-    this.updateReportDir();
-    this.renderTabHash();
+    this.initialize();
```

Comment:
Then, this change is not necessary right? There is no need to put them in the same method (initialize)? Why the original one does not work?

Reply:
Removed the `initialise` method